src/app/modules/time-entries/pages/time-entries.component.ts

```diff
@@ -145,9 +146,6 @@ export class TimeEntriesComponent implements OnInit, OnDestroy {
     if (this.entryId) {
       event.entry.id = this.entryId;
       this.store.dispatch(new entryActions.UpdateEntry(event.entry));
-      if (event.shouldRestartEntry) {
-        this.store.dispatch(new entryActions.RestartEntry(event.entry));
-      }
```

Comment:
I'm not sure why we remove the restart. Not an issue in the PR, but I would like to understand why we removed this :)